packages/edit-site/src/components/editor/global-styles-provider.js

```diff
@@ -14,7 +14,6 @@ import {
 	useMemo,
 } from '@wordpress/element';
 import { useEntityProp } from '@wordpress/core-data';
-import { __EXPERIMENTAL_STYLE_PROPERTY as STYLE_PROPERTY } from '@wordpress/blocks';
```

Comment:
If global styles does not use __EXPERIMENTAL_STYLE_PROPERTY from @wordpress/blocks I guess the need to have this data in '@wordpress/blocks' is not present anymore as we now don't reuse this data between block-editor and global styles?
We still have the need to have this metadata on the client and server. What's the advantage of passing it on the server to edit-site especially but continue to use the hardcoded client information on the block-editor?

Reply:
I think we should remove that and use this setting! Didn't do it in this PR because it felt too much.